packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -0,0 +1,169 @@
+<script lang="ts" setup>
+export type ButtonProps = {
+  label: string;
+  isDisabled?: boolean;
+  type?: "button" | "submit" | "reset";
+  color?: "primary" | "secondary" | "danger";
+  variant?: "default" | "outline" | "plain";
+};
+
+const props = withDefaults(defineProps<ButtonProps>(), {
+  label: "",
+  isLoading: false,
+  isDisabled: false,
+  type: "button",
+  color: "primary",
+  variant: "default",
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the button is clicked
+   */
+  click: [];
+}>();
+</script>
+
+<template>
+  <div
+    tabindex="0"
+    class="onyx-button"
+    :class="{
+      [`onyx-button--${props.color}`]: true,
+      [`onyx-button--${props.variant}--${props.color}`]: true,
+    }"
+  >
+    <button :disabled="props.isDisabled" class="onyx-button__element" @click="emit('click')">
+      <span class="onyx-button__element__label">{{ props.label }}</span>
+    </button>
+  </div>
+</template>
+
+<style lang="scss">
+.onyx-button {
+  display: inline-flex;
+  padding: var(--onyx-spacing-2xs, 0.5rem) var(--onyx-spacing-sm, 0.75rem);
+  justify-content: center;
+  align-items: center;
+  gap: var(--onyx-spacing-4xs, 0.25rem);
+  border-radius: var(--onyx-radius-sm, 0.25rem);
+
+  --background-color: transparent;
+  --background-hover-color: var(--onyx-color-base-primary-100, #edf4fa);
```

Comment:
All --onyx CSS variables don't need the fallback values. They are included when copying from Figma but we can remove them since all those Variables always exist.

```suggestion
  --background-hover-color: var(--onyx-color-base-primary-100);
```

Reply:
I removed the fallbacks, works fine except for the text color. I have this color in Figma: --onyx-color-text-icons-neutral-intense, and for some reason we don't have it in the code. Is the problem in Figma or in the code?

Reply:
I think Jannick renamed it in Figma but we did not yet change it in the code. Until then you can use
`var(--onyx-color-text-neutral-intense)`